test/puppeteer/index.test.ts

```diff
@@ -0,0 +1,15 @@
+// Imports
+import path from 'path';
+
+// Tests
```

Comment:
Technically this comment should be:

```suggestion
// Setup work before each test
```